packages/test/mocha-test-setup/api-report/mocha-test-setup.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="mocha" />
```

Comment:
Why was this change needed? Just want to make sure that this isn't inadvertent

Reply:
That's what prettier decided to change for some reason. I'm not sure how it ended up merged in if it has to pass prettier. Maybe something has changed in its config